### PR TITLE
Call configure only if defaultApp is null

### LINKED ISSUE #58: iOS: Default app has already been configured.
`*** Terminating app due to uncaught exception 'com.firebase.core', reason: 'Default app has already been configured.'`

Had to comment out line 49 in AppDelegate+FCMPlugin.m:
```
// [START configure_firebase]
[FIRApp configure];
// [END configure_firebase]
```

### PR DESCRIPTION
**Pull request by jcesarmobile**
In some cases the app might be already configured and calling configure twice crash the app with this error

`*** Terminating app due to uncaught exception 'com.firebase.core', reason: 'Default app has already been configured.`

Closes #58

### CODE REVIEW

**Review by ggobea — APPROVED** [review]

**Comment by dimitri320** [comment]
I'm getting this error, how can I fix it???

**Comment by jcesarmobile** [comment]
You can apply this change to your plugin, or wait until it's merged.

**Comment by dimitri320** [comment]
@jcesarmobile how can I apply this change to my plugin? Thanks!

**Comment by sahyun1** [comment]
@dimitri320  like this
cordova plugin add cordova-plugin-fcm@https://github.com/fechanique/cordova-plugin-fcm.git#0efe2150e593f1f30079e8bcd2a26cab6502e64f --save

**Comment by sahyun1** [comment]
I've tried this and the error is gone but actually message doesn't come through on ios.

**Comment by ChristianOrrala** [comment]
@sahyun1  for now I'm using [@ionic-native/firebase](https://ionicframework.com/docs/native/firebase/) with [angular/angularfire2](https://github.com/angular/angularfire2) and it's works perfectly on android and iOs.

**Comment by sahyun1** [comment]
@ChristianOrrala 
cool, so both analytics and messaging work?

**Comment by ChristianOrrala** [comment]
@sahyun1 Yes, actually with that combination I'm using Auth, Firestorage, Analytics and FCM.

**Comment by saberprashant** [comment]
Hi @sahyun1 , did you find a way to get the push notifications on iOS after solving this issue ?:
`Call configure only if defaultApp is null`

As you said, after doing this:
`
if([FIRApp defaultApp] == nil) {
        [FIRApp configure];
    }
`
I am facing the same problem where my notifications just got stopped on iOS. Please help :)

PS: I am using `cordova-plugin-fcm-with-dependecy-updated` with `cordova-plugin-firebase-analytics`

**Comment by sahyun1** [comment]
@saberprashant 
I've been using firebase plugin
https://ionicframework.com/docs/native/firebase

authentication, messaging and analytics work all fine as @ChristianOrrala suggested.

**Pull request closed without merging**
